integration/setup/config.go

```diff
@@ -0,0 +1,81 @@
+package setup
+
+import (
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/e2e-harness/pkg/framework/resource"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+)
+
+type Config struct {
+	Host       *framework.Host
+	HelmClient *helmclient.Client
+	Logger     micrologger.Logger
+	Resource   *resource.Resource
+}
+
+func NewConfig() (Config, error) {
+	var err error
+
+	var logger micrologger.Logger
+	{
+		c := micrologger.Config{}
+		logger, err = micrologger.New(c)
+		if err != nil {
+			return Config{}, microerror.Mask(err)
+		}
+	}
+
+	var host *framework.Host
+	{
+		c := framework.HostConfig{
+			Logger: logger,
+
+			ClusterID:       "n/a",
+			VaultToken:      "n/a",
```

Comment:
That is so stupid. We should further work on getting rid of the old framework thingy and come up with better designs and functionalities.

Reply:
Yes definitely the current framework abstractions don't fit well here.